packages/twenty-server/src/engine/api/graphql/workspace-query-runner/listeners/telemetry.listener.ts

```diff
@@ -48,6 +49,7 @@ export class TelemetryListener {
             data: {
               eventName: 'user.signup',
             },
+            sessionId: '',
```

Comment:
We could maybe create a @Session decorator to get the session during signup (from the cookie) and then pass it down to the backend events, the same way we pass userId / workspaceId?

Reply:
We can push this to a later PR